### PR TITLE
Update seeders and seed artwork

### PR DESCRIPTION
**Pull request by driftingly**
This PR updates the seeders and adds artwork to the seeded data.

The seeders have been split out into individual files to reduce complexity:
- `ThemeSeeder`
- `ThemePromptSeeder`
- `ArtworkSeeder`

The raw source data is processed a little differently now, adding the translations to each object allowing easier access to the translations after adding the main object.

The JSON data I'm using to populate the artwork does not provide an ID to use against the API. I have a lookup table to map the Drupal ID with an API ID. See `database/seeders/ArtworkSeeder.php`

### CODE REVIEW

**Review by nikhiltri — CHANGES_REQUESTED** [review]
I'm getting an error on `ArtworkSeeder.php:39` (Artwork::factory()->create) when I run `migrate:fresh --seed` on my local:

```
SQLSTATE[HY000]: General error: 1366 Incorrect integer value: 'LL' for column 'floor' 
```

You would think floors are integers but we do label the lower level with LL in the JSON. So this DB field probably needs to be a text field.

File: database/seeders/Behaviors/HasTwillSeeding.php
Comment (zachgarwood) [inline database/seeders/Behaviors/HasTwillSeeding.php:1]: Excellent idea

**Review by zachgarwood — APPROVED** [review]
Great work

**Comment by driftingly** [comment]
> I'm getting an error on `ArtworkSeeder.php:39` (Artwork::factory()->create) when I run `migrate:fresh --seed` on my local:
> 
> ```
> SQLSTATE[HY000]: General error: 1366 Incorrect integer value: 'LL' for column 'floor' 
> ```
> 
> You would think floors are integers but we do label the lower level with LL in the JSON. So this DB field probably needs to be a text field.

Thanks @nikhiltri - I made this update here: https://github.com/art-institute-of-chicago/journeymaker-cms/pull/9/commits/b3ed837a9bc93258f5f5802fc2f684b6f0ac7003

**Pull request merged**